tests/test_continuation.py

```
@@ -0,0 +1,364 @@
+    assert continued['request']['id'] == original['request']['id']
+    child_id = continued['result']['ids']['batch_orchestration_id']
+    assert child_id != source_id
+    assert store.get_orchestration(source_id) == source
+    assert store.list_checkpoints(source_id) == checkpoints
+    assert continued['continuation']['source_orchestration_id'] == source_id
+    assert store.get_orchestration(child_id)['state'] == 'consumed'
+
+
+def test_interrupted_after_three_usable_batches_runs_only_missing_and_integration(lane, monkeypatch):
+    from skodun.request_cancel import mark_event
+    repo, store, calls, failures = lane
```

Ruff (0.16.3)

[warning] Unpacked variable `failures` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)